lib/MusicBrainz/Server/Report/ReleasesWithoutDiscIds.pm

```diff
@@ -0,0 +1,45 @@
+package MusicBrainz::Server::Report::ReleasesWithoutDiscIds;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport';
+
+sub query {
+    q{
+        SELECT DISTINCT ON (r.id)
+            r.id AS release_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM
+        ( SELECT r.id, r.artist_credit, r.name
+          FROM release r
+          WHERE id IN (
+              SELECT release FROM medium
+              WHERE (SELECT has_discids FROM medium_format WHERE id = format)
+              AND id NOT IN (SELECT medium FROM medium_cdtoc)
+              AND status NOT IN (3, 4) -- ignore psuedo and bootleg releases
+          )
+        ) r
+    }
+}
+
+__PACKAGE__->meta->make_immutable;
+1;
+
+=head1 COPYRIGHT
```

Comment:
Please use the lighter and up-to-date section [`COPYRIGHT AND LICENSE`](https://musicbrainz.org/doc/Development/Server_Coding_Style#Perl_Module_Template_and_Documentation_Guidelines) instead.